framework/src/modules/chain/logic/account.js

```diff
@@ -446,16 +445,6 @@ class Account {
 							);
 						}
 
-						if (updatedField === 'balance') {
```

Comment:
can we already remove this? I thought `reward` part still goes through this path

Reply:
You are right. :( I will add this logic again. But this means that as long as we don't remove the transactions implemented in the core, we will have a different delegates list.